---
title: GO Subset Guide
permalink: /docs/go-subset-guide/
---

# GO Subset Guide

GO subsets (also known as GO slims) are cut-down versions of the GO ontologies containing a subset of the terms in GO. They give a broad overview of the ontology content without the detail of the specific fine-grained terms.

## How are GO subsets used?

GO subsets are particularly useful for giving a summary of the results of GO annotation of a genome, microarray, or cDNA collection when broad classification of gene product function is required. Some groups annotate to GO subsets that are relevant to their domain of interest, rather than using the full GO.
## Who creates and maintains GO subsets?

GO subsets are created by users according to their needs, and may be specific to species or to particular areas of the ontologies. GO provides a generic GO subsets which, like the GO itself, is not species-specific, and which should be suitable for most purposes. Alternatively, users can create their own GO subsets or use one of the model organism-specific subsets integrated into GO. Please email the GO helpdesk for more information about creating and submitting your GO subsets.
## GO subsets available

## Maintained GO subsets

The GO subsets in this list are maintained as part of the GO flat file. The files available below for download are generated by script from that file.
### Maintained GO subsets for download
|Organism or Usage |	Download|
|------------------|----------|
|GO slim AGR Developed by GO Consortium for the [Alliance of Genomes Resources](https://www.alliancegenome.org/)Â |[OBO format](http://current.geneontology.org/ontology/subsets/goslim_agr.obo)|
|Generic GO subset Developed by GO Consortium |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_generic.obo)|
|Plant subset Developed by [The Arabidopsis Information Resource](https://www.arabidopsis.org/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_plant.obo)|
|Candida albicans Developed by [Candida Genome Database](http://www.candidagenome.org/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_candida.obo)|
|Protein Information Resource subset Developed by Darren Natale, [PIR](https://pir.georgetown.edu/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_pir.obo)|
|Schizosaccharomyces pombe subset Developed by Val Wood, [PomBase](https://www.pombase.org/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_pombe.obo)|
|Yeast subset Developed by [Saccharomyces Genome Database](https://www.yeastgenome.org/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_yeast.obo)|
|Aspergillus subset Developed by [Aspergillus Genome Data](http://www.aspgd.org/) |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_aspergillus.obo)|
|Metagenomics subset Developed by Jane Lomax and the [InterPro](http://www.ebi.ac.uk/interpro/) group |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_metagenomics.obo)|
|Virus subset Developed by Jane Lomax and Rebecca Foulger |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_virus.obo)|
|Chembl Drug Target subset developed by Prudence Mutowo and Jane Lomax |	[OBO format](http://www.geneontology.org/ontology/subsets/goslim_chembl.obo)|
## For internal checking purposes we also provide two "anti-slims"
|Organism or Usage |	Download|
|------------------|----------|
|[Do not annotate](http://geneontology.org/ontology/subsets/gocheck_do_not_annotate.obo)|the set of high level terms that are useful for grouping, but should have no direct annotations|
|[Do not manually annotate](http://geneontology.org/ontology/subsets/gocheck_do_not_manually_annotate.obo)|the set of high level terms that are useful for grouping, but should have no direct annotations except from automated tools|

## Map2Slim option in OWLTools

Given a GO subset file, and a current ontology (in one or more files), the Map2Slim script will map a gene association file (containing annotations to the full GO) to the terms in the GO subset. This script is an option of OWLTools, and it can be used to either create a new gene association file, which contains the most pertinent GO slim accessions, or in count-mode, in which case it will give distinct gene product counts for each subset term.

Background information and details on how to download, install, and implement OWLTools, as well as instructions on how to run the Map2Slim script are available from the OWLTools Wiki at https://github.com/owlcollab/owltools/wiki/Map2Slim.
## On the web

Similarly, there are a couple of online tools that may be of use. The first is the [Princeton slimming tool](http://go.princeton.edu/), the second is the legacy [AmiGO slimmer](http://amigo1.geneontology.org/cgi-bin/amigo/slimmer). It should be noted that online tools do often contain limitations and timeouts.
