---
title: The Gene Ontology Project
permalink: /docs/about/
---

# The Gene Ontology Project

The Gene Ontology (GO) project is a major bioinformatics initiative to develop a computational representation of our
evolving knowledge of how genes encode biological functions at the molecular, cellular and tissue system levels.
Biological systems are so complex that we need to rely on computers to represent this knowledge. The project has
developed formal ontologies that represent over 40,000 biological concepts, and are constantly being revised to
reflect new discoveries. To date, these concepts have been used to "annotate" gene functions based on experiments
reported in over 140,000 peer-reviewed scientific papers. An overview of the GO resource [can be found here](/docs/introduction-to-go-resource/).

The GO resource plays an essential role in supporting biomedical research, including interpretation of large scale
molecular experiments and computational analysis of biological knowledge. It has been used in tens of thousands of
scientific studies.

The Gene Ontology Consortium (GOC) includes groups from around the world that collaborate closely on the development
of the Gene Ontology and creation of gene function annotations. [Contributors to the project are listed here](/docs/contributors/).

### GO Co-directors 
GOC is co-directed by (in alphabetical order):

+ <a href="https://www.jax.org/research-and-faculty/faculty/judith-blake">Judith Blake</a>, Jackson Laboratory (mouse gene annotation, ontology development)
+ <a href="https://med.stanford.edu/profiles/j-michael-cherry">J. Michael Cherry</a>, Stanford University (yeast gene annotation, data production processes)
+ <a href="http://www.berkeleybop.org/people/suzanna-lewis/">Suzanna Lewis</a>, Lawrence Berkeley National Laboratory (GOC software development, ontology development)
+ <a href="http://wormlab.caltech.edu/">Paul Sternberg</a>, Caltech (C. elegans gene annotation, Common Annotation Framework development)
+ <a href="https://sites.google.com/usc.edu/thomaslab">Paul Thomas</a>, University of Southern California (phylogenetic annotation, ontology development)

### The GO Council 
The GO Council consists of the heads of major participating groups (in alphabetical order):

+ <a href="http://www.ebi.ac.uk/about/people/alex-bateman">Alex Bateman</a>, UniProt, European Bioinformatics Institute (gene annotation)
+ <a href="https://www.pdn.cam.ac.uk/directory/nick-brown">Nick Brown</a>, FlyBase, University of Cambridge (Drosophila gene annotation)
+ <a href="http://fsmweb.northwestern.edu/faculty/facultyProfile.cfm?xid=10466">Rex Chisholm</a>, DictyBase, Northwestern University (Dictyostelium gene annotation)
+ <a href="https://biochemistry.tamu.edu/people/hu-james-c/">James Hu</a>, Texas A&M (gene annotation)
+ <a href="https://www.arabidopsis.org/about/staff.jsp">Eva Huala</a>, The Arabidopsis Information Resource, Phoenix Bioinformatics (Arabidopsis gene annotation)
+ <a href="http://www.ebi.ac.uk/about/people/helen-parkinson">Helen Parkinson</a>, Ontologies Team lead, European Bioinformatics Institute (ontology development)
+ <a href="http://zfin.org/ZDB-PERS-960805-676">Monte Westerfield</a>, University of Oregon (zebrafish gene annotation)

### Funding
The GO Consortium is funded by the National Human Genome Research Institute (US National Institutes of Health), grant number <a href="https://projectreporter.nih.gov/project_info_details.cfm?aid=9209989">[HG002273]</a>.
